client/client_test.go

```diff
@@ -341,7 +341,7 @@ func testFrontendImageNaming(t *testing.T, sb integration.Sandbox) {
 		},
 		ExporterDocker: func(out, imageName string, exporterResponse map[string]string) {
 			require.Contains(t, exporterResponse, "image.name")
-			require.Equal(t, exporterResponse["image.name"], "docker.io/library/"+imageName)
+			require.Equal(t, exporterResponse["image.name"], imageName)
```

Comment:
Why did this change?

Reply:
so assert was complaining that expected value is "image-docker-fe:latest". I am clueless where to change to pass this test. Any pointers?